iib/web/models.py

```diff
@@ -389,9 +392,49 @@ class Batch(db.Model):
     """A batch associated with one or more requests."""
 
     id = db.Column(db.Integer, primary_key=True)
+    _annotations = db.Column('annotations', db.Text, nullable=True)
 
     requests = db.relationship('Request', foreign_keys=[Request.batch_id], back_populates='batch')
 
+    @hybrid_property
```

Comment:
This is new to me. From what I can read in the [docs](https://docs.sqlalchemy.org/en/13/orm/extensions/hybrid.html) this is meant to translate the function body into SQL statements. Why is that needed here instead of a plain `@property`?